### PR TITLE
Remove ability to configure Feign Retryer. Fixes gh-2330

### PR DESCRIPTION
**Pull request by eacdy**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@eacdy Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-netflix&pullRequestId=2344)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-netflix&pullRequestId=2344) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@eacdy Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-netflix&pullRequestId=2344)!

**Comment by spencergibb** [comment]
See https://github.com/spring-cloud/spring-cloud-netflix/issues/2330#issuecomment-338355211

File: spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientFactoryBean.java [hunk 1]
```diff
@@ -16,9 +16,11 @@
 
 package org.springframework.cloud.netflix.feign;
 
-import java.util.Map;
-import java.util.Objects;
-
+import feign.*;
```
Comment (spencergibb) [inline spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientFactoryBean.java]: No star imports please

File: spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientFactoryBean.java [hunk 2]
```diff
@@ -30,17 +32,8 @@
...
-import feign.Feign;
-import feign.Logger;
-import feign.Request;
-import feign.RequestInterceptor;
-import feign.Retryer;
-import feign.Target.HardCodedTarget;
-import feign.codec.Decoder;
-import feign.codec.Encoder;
-import feign.codec.ErrorDecoder;
+import java.util.Map;
```
Comment (spencergibb) [inline spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientFactoryBean.java]: import order is wrong

**Comment by spencergibb** [comment]
This needs to be reopened in spring-cloud-openfeign

**Pull request closed without merging**